angular.json

```diff
@@ -105,14 +105,11 @@
           }
         },
         "lint": {
-          "builder": "@angular-devkit/build-angular:tslint",
+          "builder": "@angular-eslint/builder:lint",
```

Comment:
I don't know how to use lint or how it actually works, besides understanding what it's meant for. Can you tell me when it will run and what it will do?

I think it's a great addition by the way...!